proxy/debian.postinst

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+systemctl stop cc-proxy.socket
+systemctl stop cc-proxy.service
+[ -e /usr/lib/systemd/system/cc-proxy.service ] && rm /usr/lib/systemd/system/cc-proxy.service || true
```

Comment:
I'd remove `|| true` and change the `rm` commands to `rm -f` (but see comment above).